path/to/dgx_spark_sm121_audit.py

```
@@ -0,0 +1,41 @@
+import os
+import subprocess
+
+def check_cuda_version():
+    """
+    Check CUDA version installed on the system.
+    """
+    try:
+        output = subprocess.check_output(["nvcc", "--version"]).decode("utf-8")
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)